ui/app/components/sidebar/frame.hbs

```diff
@@ -1,42 +1,44 @@
-<HcAppFrame::Frame @hasSidebar={{@showSidebar}} as |Frame|>
+<Hds::AppFrame @hasSidebar={{@showSidebar}} as |Frame|>
```

Comment:
Since you're not using the `Frame.Header` and `Frame.Footer`, you can provide the two parameters  `@hasHeader={{false}}` and `@hasFooter={{false}}`, so the HTML tag `<header>` and `<footer>` are not generated (in any case their size is zero, but it's better for accessibility and semantic to avoid to generate them if not used).

(For docs see here: https://helios.hashicorp.design/layouts/app-frame#component-api)